treeherder/autoclassify/detect_intermittents.py

```diff
@@ -1,8 +1,8 @@
 import logging
```

Comment:
Yes but that's not at all the same as code we're running in production. I'm not going to block review on this, but IMO you should just remove these files as they're apt to confuse people. If it turns out that we want to resurrect this code, it's easy to go back in history.